### ISSUE TITLE
Bug: Character Moves Backwards/Upwards When Holding a Full Cart of Items

### ISSUE THREAD

**Issue by NOS73** [WhiteSpike/Shopping-Cart#4]
I am using the shoping cart on non-mod maps. There is a chance whenever I put items in the cart and the cart will force me move diagonally backwards or vertically upwards. 
The speed is not fast but it can phase me through the wall or railings and fall to death. 
When I clear all items in the cart it will back to normal again. But if the bug happened on planet, it will keep happening until I am back to orbit.
I currently don't have any footage yet but I'll try to record it if I see that again.

**Comment by WhiteSpike** [comment]
I need the mod list to know if there are any deprecated mods you are using. You are describing it as it always happen meanwhile I never got it since the creation of the item.

And also the logs of when that happens. I am tired of guessing what the issue is when people don't post them.

**Comment by NOS73** [comment]
Yes I am using a few deprecated mod because I still havent find any to replace them.
These are those marked deprecated:
ItemQuickSwitch: By [vasanex](https://thunderstore.io/c/lethal-company/p/vasanex/)
Matty Fixes Experimental: By [mattymatty](https://thunderstore.io/c/lethal-company/p/mattymatty/)
HDLethalCompany: By [Sligili](https://thunderstore.io/c/lethal-company/p/Sligili/)

I'll include the log when the next time it happens.

**Comment by WhiteSpike** [comment]
Closing issue due to inactivity.
